pyproject.toml

```
@@ -4,7 +4,7 @@ build-backend = "poetry.masonry.api"
 
 [tool.poetry]
 name = "redash-stmo"
-version = "2019.6.1"
+version = "2019.6.27"
```

```suggestion
version = "2019.6.2"
```

The [CalVer](https://calver.org/) format is YYYY.M.PATCH (where PATCH is a running number).

---

Ah. Thanks for the heads up!